components/Comments/Comment.vue

```diff
@@ -168,6 +229,17 @@
         remove: 'comments/remove',
         patch: 'comments/patch'
       }),
+      maxDepth () {
```

Comment:
Use the component data for static date, methods are for actions not for data.

Reply:
I moved this method to `computedMethods`